assets/section-email-signup-banner.css

```diff
@@ -18,6 +29,29 @@
   transform: translateX(-50%);
 }
 
+@media only screen and (min-width: 750px) {
+  .banner--desktop-transparent .email-signup-banner__box--no-image > *, 
+  .banner--desktop-transparent .email-signup-banner__box--no-image .field__label,
+  .banner--desktop-transparent .email-signup-banner__box--no-image .field__input {
+    color: rgb(var(--color-base-text));
+  }
+
+  .banner--desktop-transparent .email-signup-banner__box--no-image .field__input {
+    box-shadow: 0 0 0 0.1rem rgba(var(--color-base-text), 0.55);
+  }
+
+  .banner--desktop-transparent .email-signup-banner__box--no-image .field__input:focus {
+    box-shadow: 0 0 0 0.2rem rgba(var(--color-base-text), 0.75);
+  }
+
+  .banner--desktop-transparent .email-signup-banner__box--no-image .field__button:focus-visible {
+    outline: 0.2rem solid rgba(var(--color-base-text), 0.5);
+    outline-offset: 0.3rem;
+    box-shadow: 0 0 0 0.3rem rgb(var(--color-background)),
+      0 0 0.5rem 0.4rem rgba(var(--color-base-text), 0.3);
+  }
+}
```

Comment:
I'm seeing an issue on mobile when image + container are not selected. 
Same as desktop, the content isn't visible. 

<img width="786" alt="Screen Shot 2021-12-03 at 1 34 02 PM" src="https://user-images.githubusercontent.com/26778637/144654889-5668cae9-a1ad-46d4-ba02-24aae7e84c6b.png">